phoebe/frontend/default_bundles/default_binary.bundle

```diff
@@ -426,9 +426,13 @@ null
 "description": "Source for bolometric limb darkening coefficients (used only for irradiation; 'auto' to interpolate from the applicable table according to the 'atm' parameter, or the name of a specific atmosphere table)",
 "choices": [
 "auto",
+"tmap_DA",
 "phoenix",
 "tmap_sdO",
-"ck2004"
+"tmap_DO",
+"ck2004",
+"tremblay",
+"tmap_DAO"
```

Comment:
were the tmap ones just overlooked in the past?

Reply:
I think we just never rebuilt the default bundles in this particular branch.